C/manual/ch_Reports.xml

```diff
@@ -3843,81 +4015,83 @@ Income (type INCOME)
 
           <itemizedlist>
             <listitem>
-              <para><emphasis>Show budget</emphasis> - include the budgeted values for the period
+              <para><emphasis>Show budget</emphasis> &minus; include the budgeted values for the period
```

Comment:
Do you think you can convert the itemzedlist into a glosslist and drop the `-`? It seems the original author did not know variablelist or glosslist and used it as separator between the term and its explanation.

Reply:
@fellen, I think you're driving massive overuse of [`<glosslist>`](https://tdg.docbook.org/tdg/4.5/glosslist). As the docs say, "Using a [GlossList](https://tdg.docbook.org/tdg/4.5/glosslist) in running text, instead of a [VariableList](https://tdg.docbook.org/tdg/4.5/variablelist), for example, maintains the semantic distinction of a Glossary. This distinction may be necessary if you want to automatically point to the members of the list with [GlossTerm](https://tdg.docbook.org/tdg/4.5/glossterm)s in the body of the text."  A [glossary](https://www.merriam-webster.com/dictionary/glossary) is "a collection of textual [glosses](https://www.merriam-webster.com/dictionary/glosses) or of specialized terms with their meanings". That isn't even remotely related to what you're trying to use it for.

Reply:
Well, this is my first review of a report. While in dialogs glosslists make sense because often one control depends on others. in reports the [`<variablelist>`](https://tdg.docbook.org/tdg/4.5/variablelist-x) might be sufficient.

Reply:
["variablelist — A list in which each entry is composed of a set of one or more terms and an associated description"](https://tdg.docbook.org/tdg/4.5/variablelist.html) (Heh, complete with a misused emdash!)
That's no more applicable than `glosslist`, not to dialogs and not to reports. 

Plain old `itemizedlist` is generally the right choice, but only so far as a list is appropriate. E.g.
```
<section>
<title>Assets</title>
<para>The first section of the report lists the asset accounts, displayed hierarchically, with their balances projected to the end of the budget period. At the end of the section &app; will print the following totals:
<itemizedlist>
<listitem><title>Existing assets</title><para>Sum of all selected account balances at the beginning of the budget period. This is what a Balance Sheet Report would report as total assets for the same selected accounts on that date.</para>
<!-- … --> 
</itemizedlist>
</para>
</section>
```
While `glosslist` and `variablelist` assign the wrong semantic meaning in this case, even trying to wrap bits of documentation in semantic tags is a relic of the 1990s. The semantic web fantasy didn't pan out. Stop wasting everyone's time and just focus on getting the right words in place with a reasonable layout.

Reply:
Nice idea, but
```
C/manual/index.docbook:3792: element listitem: validity error : Element listitem content does not follow the DTD, expecting (calloutlist | glosslist | bibliolist | itemizedlist | orderedlist | segmentedlist | simplelist | variablelist | caution | important | note | tip | warning | literallayout | programlisting | programlistingco | screen | screenco | screenshot | synopsis | cmdsynopsis | funcsynopsis | classsynopsis | fieldsynopsis | constructorsynopsis | destructorsynopsis | methodsynopsis | formalpara | para | simpara | address | blockquote | graphic | graphicco | mediaobject | mediaobjectco | informalequation | informalexample | informalfigure | informaltable | equation | example | figure | table | msgset | procedure | sidebar | qandaset | task | anchor | bridgehead | remark | highlights | abstract | authorblurb | epigraph | indexterm | beginpage)+, got (title para )
Document /home/frank/git/gnucash-docs-old/C/manual/index.docbook does not validate
```
Normal lists allow no title in their elements, only for the whole list.  That is the reason why we use lists with term and definition part. I assume we still want get rid of constructs like
 `<emphasis role="bold">Term</emphasis>, a changing combination of colons or dashs and spaces; Definition`